src/opnsense/mvc/app/models/OPNsense/OpenVPN/OpenVPN.php

```diff
@@ -600,7 +600,7 @@ public function generateInstanceConfig($uuid = null)
                             $options['ifconfig'] = "{$ip1} {$ip2}";
                             $options['ifconfig-pool'] = "{$ip2} {$ip3}";
                         } else {
-                            $options['server'] = $parts[0] . " " . $mask;
+                            $options['server'] = $parts[0] . " " . $mask . ($node->nopool == '1' ? ' nopool' : '');
```

Comment:
```suggestion
                            $options['server'] = $parts[0] . " " . $mask;
                            if ($node->nopool->isEqual('1')) {
                                $options['server'] .=  ' nopool';
                            }
```

something like this is easier to maintain and avoids implicit casting